data/src/main/java/io/gnosis/data/models/ChainInfo.kt

```diff
@@ -4,12 +4,14 @@ import com.squareup.moshi.Json
 import com.squareup.moshi.JsonClass
 import pm.gnosis.common.adapters.moshi.DecimalNumber
 import java.math.BigInteger
+import io.gnosis.contracts.BuildConfig.ENS_REGISTRY
 
 
 @JsonClass(generateAdapter = true)
 data class ChainInfo(
     @Json(name = "chainId") @field:DecimalNumber val chainId: BigInteger,
     @Json(name = "chainName") val chainName: String,
+    @Json(name = "ensRegistryAddress") val ensRegistryAddress: String? = ENS_REGISTRY,
```

Comment:
this is probably wrong. We can't assume default ens registry would work on all chains

Reply:
This should not be there anymore. It was only for testing.